Makefile

```diff
@@ -108,7 +118,14 @@ docker:
 		.
 	@echo "Built Docker image \"$(IMAGE_NAME)\""
 
-# Run the built cni container image to use in functional testing
+init:
```

Comment:
can we call this target `docker-init` or `docker-build-init-image` to align with the other docker build-related targets?

Reply:
Renamed the target to `docker-init` since I had to rebase anyway.